config/routes.rb

```diff
@@ -1144,56 +1144,62 @@ def patch(*)
 
     %w[groups g].each do |root_path|
       resources :groups,
-                only: %i[index show new edit update],
+                only: %i[index new edit update],
                 id: RouteFormat.username,
```

Comment:
Is this constraint still applied now that the parameter has been changed to `:name`?